com.usebottles.bottles.dev.yml

```diff
@@ -273,6 +258,30 @@ modules:
       - type: archive
         url: https://github.com/ishitatsuyuki/LatencyFleX/releases/download/v0.1.0/latencyflex-v0.1.0.tar.xz
         sha256: 9862c7608b85ac6bff018d5926c4692490ff84ca3e649119b7fc53680e64dd1a
+  
+  - name: libadwaita
+    buildsystem: meson
+    config-opts:
+    - -Dexamples=false
+    - -Dtests=false
+    sources:
+    - type: git
+      url: https://gitlab.gnome.org/GNOME/libadwaita.git
+      commit: 39a9a4b5b5987737537d6a044da7c7fdb91f544f
```

Comment:
This is 1.0 though?

Reply:
1.0.4

Reply:
we can switch to `5ec9f624b23bf78b29dd708e77ccdfcee0a9867a` for 1.1.1 which should be the most recent

Reply:
Either way, let's either keep the yml as is or delete it outright - we have the json manifest already

Reply:
I prefer and use yaml, it's also more clear to read

Reply:
Ok